packages/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -85,6 +86,8 @@ class GoogleMap extends StatefulWidget {
   /// Circles to be placed on the map.
   final Set<Circle> circles;
 
+  final EdgeInsets padding;
```

Comment:
can you add dart doc explaining this field?

Reply:
Added `Padding to be set on map. See https://developers.google.com/maps/documentation/android-sdk/map#map_padding for more details.`